react-native-in-app-payments-quickstart/app/service/Charge.tsx

```diff
@@ -16,7 +16,7 @@
 import { CHARGE_SERVER_URL } from '../Constants';
 import ChargeError from '../ChargeError';
 
-export default async function chargeCardNonce(nonce, verificationToken) {
+export default async function chargeCardNonce(nonce: any, verificationToken = undefined) {
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it.